### PR TITLE
Update readme file to reflect the current situation

### PR DESCRIPTION
**Pull request by breunigs**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,11 +55,7 @@ For the following features, pull requests will be accepted:
   retry when uploading a file fails.
 * Enhance the Preferences activity to allow configuration of multiple WebDAV
   servers, add a popup dialog with server selection in the ShareActivity.
-* Beautify the Preferences activity to show the current values of WebDAV
-  URL/username underneath the label (see comment in
```
Comment (stapelberg) [inline README.md:59]: Why are you removing this? This is not done.

File: README.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 DavSync (or net.zekjur.davsync to be specific)
 ==============================================
 
-<img src="https://github.com/mstap/android-davsync/raw/master//screenshots/davsync-2013-03-10-200731.png" width="240" align="right" alt="DavSync screenshot">
+<img src="https://github.com/mstap/android-davsync/raw/master/screenshots/davsync-2013-03-10-200731.png" width="240" align="right" alt="DavSync screenshot">
```
Comment (stapelberg) [inline README.md]: s/mstap/stapelberg/ while you’re at it.

File: README.md (hunk 1 above)
Comment (breunigs) [inline README.md:59]: Hm, maybe I don't understand the issue, but this is what it looks like for me in the prefs. Is this bullet point referring to something else?

Comment (breunigs) [inline README.md:59]: Ah sorry, the reply via mail function apparently ignores attached images. Here’s the screenshot I wanted to share: http://www.yrden.de/share/2014-07-21/Screenshot_2014-07-21-11-06-25.png

Could it be that the comment in the settings activity is outdated?

Comment (stapelberg) [inline README.md:59]: Indeed. commit 6c95bc4d fixed the problem, but neglected to remove the comment. So please extend your pull request to also delete the comment.

**Comment by breunigs** [comment]
I don't think I will work on this any time soon :)

**Pull request closed without merging**